python-package/xgboost/dask.py

```diff
@@ -65,18 +75,24 @@ def create_worker_dmatrix(*args):
     begin_partition = partition_size * rabit.get_rank()
     end_partition = min(begin_partition + partition_size, total_partitions)
     dmatrix_args = []
+    dmatrix_kwargs = {}
+    # Convert positional args
     for arg in args:
         if isinstance(arg, (dask.dataframe.core.DataFrame, dask.dataframe.core.Series)):

```

Comment:
You can safely drop the `.core` from this type.  That will make things slightly less brittle if we ever choose to rearrange our internal directory structure.